docs/wiki/Market-Making.md

```diff
@@ -0,0 +1,161 @@
+
+<img src="./images/using-bison-wallet/mm-placements-quick.png" width="400" alt="">
+
+The following placement settings are available in Quick Configuration mode:
+
+* **Price Levels Per Side:** The number of steps or levels at which the bot will place orders on each side of the order book.
+* **Lots per Level:** The depth of each order, defined by the number of lots the bot allocates per price level.
+* **Profit Threshold:** Adjusts the spread by adding a margin to the breakeven spread, expressed as a percentage of the mid-gap price.
+* **Price Increment:** The price difference between consecutive order levels in the order book.
+
+<img src="./images/using-bison-wallet/mm-placements-quick-layout.png" width="800" alt="">
+
+
+#### Manual Configuration
+
+Manual placement configuration allows users to fine-tune their market-making strategy with greater precision while also providing access to various algorithms for order placement.
+
+
+<img src="./images/using-bison-wallet/mm-placements-manual.png" width="400" alt="">
+
+The following placement settings are available in the Manual Configuration mode:
+
+* **Gap Strategy:** Sets the algorithm for calculating the distance from the basis price to place orders. The following strategies are available:
+    * **Percent Plus:** The gap starts at the break-even spread, which is the minimum spread at which a buy-sell combo produces profit, 
+    and then adds the specified spread, calculated as a percentage of the mid-gap price.
+    * **Percent:** The gap is set to the specified percent of the mid-gap price.
+    * **Absolute Plus:** The gap starts at the break-even spread, which is the minimum spread at which a buy-sell combo produces profit, 
+    and then adds the specified rate difference to both sides.
+    * **Absolute:** The spread is set to the specified value both left and right of the mid-gap, resulting in a total gap of twice the 
+    specified value.
+    * **Multiplier:** The gap is set to the specified multiple of the break-even spread, which is the minimum spread at which a buy-sell combo produces profit.
+
+* **Buy Placements:** Defines the bot's buy order placements. If the available balance is insufficient to place all orders, the bot will prioritize them accordingly. 
+* **Sell Placements:** Defines the bot's sell order placements. If the available balance is insufficient to place all orders, the bot will prioritize them accordingly.
```

Comment:
> If the available balance is insufficient to place all orders, the bot will prioritize them accordingly. 

You could write this separately somewhere instead of repeating. This is true for quick config as well.